lte/cloud/go/services/lte/obsidian/handlers/handlers_test.go

```diff
@@ -1006,14 +1006,10 @@ func TestCreateGateway(t *testing.T) {
 		serdes.Entity,
 	)
 	assert.NoError(t, err)
-	err = device.RegisterDevice(
-		"n1", orc8r.AccessGatewayRecordType, "hw2",
-		&models.GatewayDevice{
-			HardwareID: "hw2",
-			Key:        &models.ChallengeKey{KeyType: "ECHO"},
-		},
-		serdes.Device,
-	)
+	err = device.RegisterDevice(context.Background(), "n1", orc8r.AccessGatewayRecordType, "hw2", &models.GatewayDevice{
+		HardwareID: "hw2",
+		Key:        &models.ChallengeKey{KeyType: "ECHO"},
+	}, serdes.Device)
```

Comment:
Here and elsewhere: ideally let's preserve/make uniform the formatting of these

Or, basically the min of what I'm looking for is newline after `(` and newline before `)`, other than that I care less

Reply:
yeah this is the refactorer's doing, I can do a once-over and check